rolling/distribution.yaml

```diff
@@ -2601,7 +2601,7 @@ repositories:
     doc:
       type: git
       url: https://github.com/boschglobal/locator_ros_bridge.git
-      version: main
```

Comment:
Would these be better as https://github.com/boschglobal/locator_ros_bridge/tree/main-v1.4 ?

Reply:
Yeah, good question.  I honestly don't know.  @syyuen will have to answer.